lib/play_game.py

```diff
@@ -3,27 +3,38 @@
 
 
 class PlayGame:
+
+    current_player = "X"
+    next_player = "O"
+
     def __init__(self, board, ui):
         self.board = board
         self.ui = ui
 
     def start(self):
         self.ui.display("Welcome to TIC TAC TOE!")
         self.board.display_board()
+        self.ui.display(self.current_player + " goes first")
         while not self.board.board_full(self.board.board):
-            self.turn()
+            self.turn(self.current_player)
+            self.switch_players()
 
-    def turn(self):
+    def turn(self, mark):
```

Comment:
I also am not sure if anyone mentioned naming conventions for functions and classes yet. 

Typically your function names should be expressive of what the function is doing. In this case, `turn` would be more clear as `take_turn`. Usually the function names are going to be a verb, since they are performing an action.

The class name, on the other hand, is the representation for the object, so a noun makes more sense. So instead of `PlayGame`, which sounds more like a function name, I would just call this class `Game`.